examples/fetching_messages.py

```diff
@@ -9,10 +9,9 @@
 
 # Make sure you've imported either the mcap-ros1-support or mcap-protobuf-support
 # libraries before making this call in order to get decoded messages.
-messages = client.get_messages(
+for schema, channel, message, decoded_message in client.iter_messages(
     device_id=device_id,
     start=datetime.now() - timedelta(hours=3),
     end=datetime.now() - timedelta(hours=1),
-)
-
-print(f"downloaded {len(messages)} messages")
+):
+    print(f"[{message.log_time}] Channel: {channel.topic} Schema: {schema.name}")
```

Comment:
nit: formatting of the labels feels a bit hard to read for me:

```
[1479512770391847576] Channel: /imu/data Schema: sensor_msgs/Imu
[1479512770391861165] Channel: /time_reference Schema: sensor_msgs/TimeReference
[1479512770391922289] Channel: /time_reference Schema: sensor_msgs/TimeReference
[1479512770391937993] Channel: /pressure Schema: sensor_msgs/FluidPressure
```

Maybe something simpler?

```suggestion
    print(f"{message.log_time} | {channel.topic} \t| {schema.name}")
```

i.e.


```
1479512810369905916 | /fix	| sensor_msgs/NavSatFix
1479512810369936841 | /ecef/	| geometry_msgs/PointStamped
1479512810370480491 | /time_reference	| sensor_msgs/TimeReference
1479512810370530865 | /time_reference	| sensor_msgs/TimeReference
1479512810370682589 | /imu/data	| sensor_msgs/Imu
1479512810370709498 | /time_reference	| sensor_msgs/TimeReference
1479512810370745760 | /fix	| sensor_msgs/NavSatFix
```